addons/account/models/account_journal_dashboard.py

```diff
@@ -420,6 +420,7 @@ def _fill_bank_cash_dashboard_data(self, dashboard_data):
             has_outstanding, outstanding_pay_account_balance = outstanding_pay_account_balances[journal.id]
             to_check_balance, number_to_check = to_check.get(journal, (0, 0))
             misc_balance, number_misc = misc_totals.get(journal.default_account_id, (0, 0))
+            currency_consistency = not journal.currency_id or journal.currency_id == journal.default_account_id.currency_id
```

Comment:
```suggestion
            currency_consistent = not journal.currency_id or journal.currency_id == journal.default_account_id.currency_id
```
can you rename this (and below too)

otherwise it's good by me